SJH/Day3/K번째수.js

```diff
@@ -0,0 +1,19 @@
+function solution( array, commands ) {
+    let answer = [];
+    
+    for ( let x = 0; x < commands.length; x++ ) {
+        let i = commands[x][0];
+        let j = commands[x][1];
+        let k = commands[x][2];
```

Comment:
```suggestion
        let [i, j, k] = commands[x];
```
자바스크립트의 구조분해 할당을 이용하면 이렇게 줄일 수 있습니다!

Reply:
구조분해 할당을 활용하니 코드가 간단해져 읽기 좋은 것 같습니다~